languages/minecraft/acf-minecraft_en.properties

```diff
@@ -32,4 +32,3 @@ acf-minecraft.is_not_a_valid_name = Error: <c2>{name}</c2> is not a valid userna
 acf-minecraft.multiple_players_match = Error: Multiple players matched <c2>{search}</c2> <c3>({all})</c3>, please be more specific.
 acf-minecraft.no_player_found_server = No player matching <c2>{search}</c2> is connected to this server.
 acf-minecraft.no_player_found = No player matching <c2>{search}</c2> could be found.
-
```

Comment:
I know, but you had two ;)
github is being weird at rendering it, but there is a CRLF behind the last line